examples/plot_decoding_tutorial.py

```diff
@@ -78,41 +97,76 @@
 # voxels in the mask
 print(fmri_masked.shape)
 
+###########################################################################
+# One way to think about what just happened is to look at it visually:
+#
+# .. image:: /images/masking.jpg
+#
+# Essentially, we can think about overlaying a 3D grid on an image. Then, 
+# our mask tells us which cubes or "voxels" (like 3D pixels) to sample from. 
+# Since our Nifti images are 4D files, we can't overlay a single grid -- 
+# instead, we use a series of 3D grids (one for each volume in the 4D file), 
+# so we can get a measurement for each voxel at each timepoint. These are 
+# reflected in the shape of the matrix ! You can check this by checking the 
+# number of positive voxels in our brain mask.
+#
+# .. seealso::
+# 	There are many other strategies in Nilearn `for masking data and for
+# 	generating masks 
+# 	<https://nilearn.github.io/manipulating_images/manipulating_images.html#computing-and-applying-spatial-masks>`_.
+# 	I'd encourage you to spend some time exploring the documentation for these.
+# 	We can also `display this time series <https://nilearn.github.io/auto_examples/03_connectivity/plot_adhd_spheres.html#display-time-series>`_ to get an intuition of how the 
```

Comment:
I thought we got rid of the ADHD dataset throughout the examples ?

Reply:
Cannot comment on this I don't know :P
@GaelVaroquaux

Reply:
Went back through the `whats_new.rst` -- it was merged into another example and renamed as `plot_sphere_based_connectome.py` !

Reply:
There's still a `display-time-series` heading [here](https://github.com/nilearn/nilearn/blob/ab4db4a0a130061791947434df16b97b607b843d/examples/03_connectivity/plot_sphere_based_connectome.py#L81), so just updating the example name should work :smile_cat: